networkit/cpp/graph/test/GraphGTest.cpp

```diff
@@ -132,6 +132,18 @@ void GraphGTest::SetUp() {
 
 /** CONSTRUCTORS **/
 
+TEST(GraphGTest, testDefaultConstructorIndexed) {
```

Comment:
I'd recommend to split this into two tests, one for directed and one for undirected graphs.
Test isolation is something that we could improve across the whole library.

Reply:
You are right. In my mind the test was too small to seperate it, but this more a decision about how we want to structure the tests.